containerize/Containerfile

```diff
@@ -92,11 +92,12 @@ COPY ./containerize/container_default_config.yaml /opt/rapidast/rapidast-default
 COPY ./containerize/path_rapidast.sh /etc/profile.d/rapidast.sh
 
 ### Install RapiDAST requirements, globally, so that it's available to any user
-RUN microdnf install -y --setopt=install_weak_deps=0 java-11-openjdk shadow-utils dbus-glib procps git nodejs npm && \
+RUN microdnf install -y --setopt=install_weak_deps=0 java-11-openjdk shadow-utils dbus-glib procps git nodejs npm gcc python3-devel && \
```

Comment:
`gcc` and `python3-devel` are required to compile `pyyaml` when it is pre-fetched. To keep the image clean, these packages are removed after use

Reply:
I'm wondering if we could more optimize leveraging the multi-stage build?

Reply:
We could probably move the compilation to an earlier stage of the container build, but that might make things more complicated. We'd have to figure out exactly which files to copy from the compiler stage to the runtime stage. There are workarounds, like using a venv, installing dependencies locally or using wheels directly, although not sure if this works adds really much, as if we clean the data in the same container layer it shouldn't add much size to the final image. If you think it's worth investigating, I'll create a ticket for it.

Reply:
Ok. We could investigate it separately

Reply:
Finally, I've removed the build dependencies from the `Containerfile` since prefetching now retrieves the wheels as well. This eliminates the need to build or compile dependencies, allowing direct use of the wheels. While this approach increases the overall download size, it significantly speeds up the build process.